src/applications/ivc-champva/10-10D/components/Sponsor/sponsorFileUploads.js

```diff
@@ -190,48 +184,107 @@ export function uploadWithInfoComponent(
       'view:additionalResources': {
         'ui:description': (
           <>
-            <p>
-              <b>Resources regarding {category}</b>
-            </p>
-            <ul>
-              {resources &&
-                resources.map((resource, index) => (
-                  <li key={`link-${resource}-${index}`}>
-                    {makeLink(resource)}
-                  </li>
-                ))}
-            </ul>
+            {resources ? (
+              <>
+                <p>
+                  <b>Resources regarding {category}</b>
+                </p>
+                <ul>
+                  {resources.map((resource, index) => (
+                    <li key={`link-${resource}-${index}`}>
+                      {makeLink(resource)}
+                    </li>
+                  ))}
+                </ul>
+              </>
+            ) : (
+              ''
+            )}
           </>
         ),
       },
-      'view:fileUploadMessage': isOptional
-        ? { ...optionalFileUploadMessage }
-        : { ...requiredFileUploadMessage },
+      // 'view:fileUploadMessage': isOptional
+      //   ? { ...optionalFileUploadMessage }
+      //   : { ...requiredFileUploadMessage },
       ...fileUploadBlurb,
     },
     schema: {
       'view:acceptableFilesList': blankSchema,
       'view:additionalResources': blankSchema,
-      'view:fileUploadMessage': blankSchema,
+      // 'view:fileUploadMessage': blankSchema,
       'view:fileUploadBlurb': blankSchema,
     },
   };
 }
 
 export const sponsorDisabilityRatingConfig = uploadWithInfoComponent(
-  acceptableFiles.disabilityCert,
+  undefined, // acceptableFiles.disabilityCert,
```

Comment:
Temporarily circumventing the acceptable files list functionality (you'll see this wherever `uploadWithInfoComponent` is used.